convert/drone/convert.go

```diff
@@ -165,23 +158,11 @@ func (d *Converter) convert(ctx *context) ([]byte, error) {
 		case v1.KindPipeline:
 			// TODO pipeline.name removed from spec
 			// pipeline.Name = from.Name
-
-			pipeline.Stages = append(pipeline.Stages, &v2.Stage{
-				Name:     from.Name,
-				Type:     "ci",
-				When:     convertCond(from.Trigger),
-				Delegate: convertNode(from.Node),
-				Spec: &v2.StageCI{
-					Clone:    convertClone(from.Clone),
-					Envs:     copyenv(from.Environment),
-					Platform: convertPlatform(from.Platform),
-					Runtime:  convertRuntime(from),
-					Steps:    convertSteps(from, d.orgSecrets),
-					Volumes:  convertVolumes(from.Volumes),
-
-					// TODO support for delegate.selectors from from.Node
-					// TODO support for stage.variables
-				},
+			pipeline.Stages = append(pipeline.Stages, &v2.StageV1{
+				Name:    from.Name,
+				Clone:   convertCloneV1(&from.Clone),
+				Runtime: "machine",
```

Comment:
We should not hard code it

Reply:
updated the code and made Runtime dynamic.